nginx.conf

```diff
@@ -172,11 +172,16 @@
       proxy_set_header Connection "";
     }
 
-    # proxy requests to the Percona news rss feed
+    # proxy requests to the Percona's blog feed
+    # fallback to local rss if pmm-server is isolated from internet.
+    # https://jira.percona.com/browse/PMM-6153
+    root /usr/share/pmm-server;
```

Comment:
Shouldn't we put this `root` inside `location`? Or, if this is global, let's move it to the top of the config.

Reply:
I’d like to put it into location as well. Check different solution, but it works only in such way.

Reply:
We definitely should not use `root`. I think `alias` inside `location` should work. Please try it.

Reply:
Changed `root /usr/share/pmm-server` to `root /usr/share/pmm-server/static`